# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/builtin/intl/DisplayNames.js
# Commit: 38dea55d2978
# Full Hash: 38dea55d2978cfe2d15b76734747b049705fb1bb
# Author: Jeff Walden <jwalden@mit.edu>
# Date: 2020-05-28 15:51:15
# Description:
#   Bug 1640899 - Make sure to supply all three arguments to |intl_ValidateAndCanonicalizeUnicodeExtensionType| when invoking it in |Intl.DisplayNames|.  r=sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77179
# ==============================================================================

diff -r 983a55f50d60 -r 38dea55d2978 js/src/builtin/intl/DisplayNames.js
--- a/js/src/builtin/intl/DisplayNames.js	Wed May 27 23:01:15 2020 +0000
+++ b/js/src/builtin/intl/DisplayNames.js	Wed May 27 23:00:21 2020 +0000
@@ -152,7 +152,7 @@
         var calendar = GetOption(options, "calendar", "string", undefined, undefined);
 
         if (calendar !== undefined) {
-            calendar = intl_ValidateAndCanonicalizeUnicodeExtensionType(calendar, "calendar");
+            calendar = intl_ValidateAndCanonicalizeUnicodeExtensionType(calendar, "calendar", "ca");
         }
 
         opt.ca = calendar;
